### DISCUSSION TITLE
bk-pdf-viewer parameters

### DISCUSSION THREAD

**Discussion by rafpor75** [mia-platform/community#504] [General]
### Description

Hi all,
Is there a way to customize the back-kit library pdf-viewer, I have particular cases in which I am loading images to display in a popup, I load images much larger than the size of the popup window or much smaller and it would be It is useful to resize the popup window if the image is too small or resize the image otherwise

Thank you

**Comment by audeconteditilbe** [comment]
Hi @rafpor75,

thanks for reaching out!
Currently, there is no configuration available to achieve a dynamically resizing popup window using the `bk-pdf-viewer` component.
Please note that the PDF Viewer is primarily designed to interact with PDF files, as stated [in the documentation](https://docs.mia-platform.eu/docs/microfrontend-composer/back-kit/components/pdf_viewer).
